admin-dev/themes/new-theme/template/layout.tpl

```diff
@@ -20,50 +20,36 @@
-    <div class="component hidden-md-down">{include file="components/layout/search_form.tpl"}</div>
+    <div class="component d-none d-md-inline-block">{include file="components/layout/quick_access.tpl"}</div>
+    <div class="component d-none d-md-inline-block col-md-4">{include file="components/layout/search_form.tpl"}</div>
 
-
-    <div class="component pull-md-right -norightmargin hidden-md-down">{include file="components/layout/employee_dropdown.tpl"}</div>
-    {if $show_new_orders || $show_new_customers || $show_new_messages}
-      <div class="component pull-xs-right">{include file="components/layout/notifications_center.tpl"}</div>
+    {if isset($debug_mode) && $debug_mode == true}
+      <div class="component d-none d-md-inline-block">
+        <div class="shop-state" id="debug-mode">
+          <i class="material-icons">bug_report</i>
+          <span class="label-tooltip" data-toggle="tooltip" data-placement="bottom" data-html="true"
+                title="<p class='text-left text-nowrap'><strong>{l s='Your shop is in debug mode.'}</strong></p><p class='text-left'>{l s='All the PHP errors and messages are displayed. When you no longer need it, [1]turn off[/1] this mode.' html=true sprintf=['[1]' => '<strong>', '[/1]' => '</strong>']}</p>">{l s='Debug mode'}</span>
+        </div>
+      </div>
     {/if}
-    <div class="component pull-md-right hidden-md-down">{include file="components/layout/shop_list.tpl"}</div>
     {if isset($maintenance_mode) && $maintenance_mode == true}
-      <div class="component pull-right hidden-md-down">
+      <div class="component d-none d-md-inline-block">
         <div class="shop-state" id="maintenance-mode">
           <i class="material-icons">build</i>
           <a class="label-tooltip" data-toggle="tooltip" data-placement="bottom" data-html="true" title="<p class='text-left text-nowrap'><strong>{l s='Your shop is in maintenance.'}</strong></p><p class='text-left'>{l s='Your visitors and customers cannot access your shop while in maintenance mode.%s To manage the maintenance settings, go to Shop Parameters > Maintenance tab.' sprintf=['<br />']}</p>" href="{$link->getAdminLink('AdminMaintenance')|escape:'html':'UTF-8'}">
-              {l s='Maintenance mode'}
+            {l s='Maintenance mode'}
           </a>
         </div>
       </div>
     {/if}
-    {if isset($debug_mode) && $debug_mode == true}
-      <div class="component pull-right hidden-md-down">
-        <div class="shop-state" id="debug-mode">
-          <i class="material-icons">bug_report</i>
-          <span class="label-tooltip" data-toggle="tooltip" data-placement="bottom" data-html="true"
-            title="<p class='text-left text-nowrap'><strong>{l s='Your shop is in debug mode.'}</strong></p><p class='text-left'>{l s='All the PHP errors and messages are displayed. When you no longer need it, [1]turn off[/1] this mode.' html=true sprintf=['[1]' => '<strong>', '[/1]' => '</strong>']}</p>">{l s='Debug mode'}</span>
```

Comment:
Where did this block go?

Reply:
Just here https://github.com/PrestaShop/PrestaShop/pull/8278/files/0d41f0a0c1e9cd751d2bbe8ca3f221cbc07e2131#diff-a0e02ff05deebe3064e9313893ebfb3fR31